frontend/src/components/project/RadiobuttonList.vue

```diff
@@ -12,55 +12,64 @@
             </v-radio-group>
             <DatePicker
                 v-if="selectedOption === 'student'"
-                v-model="radio_date"
-                label="Group Selection Deadline"
-            ></DatePicker>
-            <!-- Numeric input for Capacity -->
+                :modelValue="enrollDate"
+                @update:modelValue="handleDateChange"
+                label="Deadline"
+            />
             <v-text-field
-                v-if="selectedOption === 'student' || selectedOption === 'random'"
                 v-model="capacity"
-                label="Capacity"
+                :label="$t('project.capacity_group')"
                 type="number"
                 min="1"
+                @input="handleCapacityInput"
             ></v-text-field>
         </v-container>
     </v-card>
 </template>
 
 <script setup lang="ts">
-import { ref, watch } from "vue";
-import DatePicker from "./DatePicker.vue"; // Import your custom DatePicker component
+import { ref, defineProps, defineEmits, watch } from "vue";
+import DatePicker from "./DatePicker.vue";
 
-interface RadioButtonOption {
-    label: string;
-    value: string;
-}
-
-const model = defineModel<string>({});
-
-defineProps<{
-    title: string;
-    options: RadioButtonOption[];
-}>();
-
-const isToggled = ref(true);
-const selectedOption = ref(model);
-const radio_date = ref(new Date());
-const capacity = ref(1); // Default capacity
+const props = defineProps({
+    title: String,
+    options: Array,
+    initialDate: Date,
+    initialCapacity: Number,
+});
```

Comment:
```suggestion
const props = defineProps<{
    title: string;
    options: any[];
    initialDate: Date;
    initialCapacity: number;
}>();
```
Bij options zou nog gespecifiëerd moeten worden wat er precies in die array zit